deepchem/models/tests/test_generalize.py

```diff
@@ -188,3 +189,93 @@ def model_builder(model_dir):
   #  scores = model.evaluate(test_dataset, [classification_metric])
   #  for score in scores[classification_metric.name]:
   #    assert score > .5
+
```

Comment:
@miaecle The way I implement XGBoost makes it hard to actually overfit a dataset. That's the reason I write unit tests on test_generalize.py